src/Conversion/ONNXToKrnl/Tensor/DepthToSpace.cpp

```diff
@@ -73,119 +71,28 @@ struct ONNXDepthToSpaceOpLowering : public ConversionPattern {
       perm = {0, 1, 4, 2, 5, 3};
     }
 
+    OnnxBuilder onnxBuilder(rewriter, loc);
```

Comment:
nit: since it will need to be rebased, MLIR suggested that the builder's name be named `create` so that it then read `create.reshape(...)`. This convention is currently used throughout the code, might be nice to adopt it here too.

Reply:
done